service/src/main/java/bio/terra/axonserver/service/cromwellworkflow/CromwellWorkflowService.java

```diff
@@ -266,4 +316,16 @@ private static List<ApiFailureMessages> toApiFailureMessages(
                     .causedBy(toApiFailureMessages(failure.getCausedBy())))
         .toList();
   }
+
+  private File createSafeTempFile(String filePrefix, String fileSuffix) throws IOException {
```

Comment:
Just an FYI once my current PR goes in we can refactor this to use the new Autoclosable class. nit: maybe add a comment to that effect

Reply:
Done, now using Autoclosable.